### DISCUSSION TITLE
Packaging and shipment cost

### DISCUSSION THREAD

**Discussion by LarsK1** [inventree/InvenTree#7743] [Q&A]
Hi everybody,
we're using InvenTree to manage our logistigs and to track our sales prices. Since we only sell through platforms like ebay and amazon, we would like to include our shippment costs as well as the fees these platforms charge. What's the best way to include them? Do you we add them as an additional product, or is there an specialized field for this kind of charges?

Thanks for your help in advance!

**Comment by SchrodingersGat** [comment]
You can add anciliary charges as "extra line items" in sales orders, if this is what you mean? Or do you want to include these "costs" in the sale price of individual units you are selling?

**Comment by LarsK1** [comment]
Hi,
I would like to include the cost in the product pricing, so that I can get the real item costs.

**Reply by SchrodingersGat** [reply]
Ok, this is not directly supported currently and would require some new features to add in extra cost items to a product. 

We need to consider how best to add "non production costs" to a part for purpose of sale price calculation. Likely this would quickly become quite complex as everyone will have different requirements. So, at bare minimum a facility for adding extra part costs, not associated with the BOM. This could then be augmented with plugins

**Reply by LarsK1** [reply]
That would be an awesome solution!

**Comment by cabottech** [comment]
Just thought I'd throw in my two cents on how we handle this sort of thing.

We have a spreadsheet that is used for each purchase order. We input the supplier's line item price per part, and then add all additional costs, such as freight, import duties, currency exchange fees, etc. This extra cost is proportionally distributed to the parts. This adjusted costing is what we then enter into InvenTree.

Why we do this:

- It effectively "bakes-in" the full cost of getting a part on to our shelf.
- It reconciles added fees or discounts specific to suppliers.
- This makes our quoting more convenient: InvenTree part cost or BOM cost + our margin.
- We are a relatively small operation, so while there is some overhead involving the spreadsheet, it simplifies everything else.

I'm interested in hearing how other's handle this.
